### PR TITLE
Add Seda chain to the registry

### PR DESCRIPTION
**Pull request by kleomedes**

### CODE REVIEW

File: seda/chain.json [hunk 1]
```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "seda",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.seda.xyz/",
+  "pretty_name": "Seda",
```
Comment (hacheigriega) [inline seda/chain.json:7]: Can we change it to "SEDA"? (all capitalized)

File: seda/chain.json [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        "id": "31f54fbcf445a9d9286426be59a17a811dd63f84",
+        "address": "18.133.231.208:26656"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://rpc.mainnet.seda.xyz/",
+        "provider": "Seda"
```
Comment (hacheigriega) [inline seda/chain.json:96]: ditto. "SEDA", please. :)

File: seda/chain.json [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+      {
+        "address": "https://seda-rpc.kleomedes.network/",
+        "provider": "Kleomedes"
+      }
+
+    ],
+    "rest": [
+      {
+        "address": "https://lcd.mainnet.seda.xyz/",
+        "provider": "Seda"
```
Comment (hacheigriega) [inline seda/chain.json:123]: ditto.

File: seda/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          "denom": "aseda",
+          "exponent": 0
+        },
+        {
+          "denom": "seda",
+          "exponent": 18
+        }
+      ],
+      "base": "aseda",
+      "name": "Seda",
```
Comment (hacheigriega) [inline seda/assetlist.json:18]: "Seda" -> "SEDA"

**Comment by hacheigriega** [comment]
Thank you so much for opening this PR @kleomedes We just pointed out capitalization of our company's name - I know it's confusing :D

**Comment by kleomedes** [comment]
> Thank you so much for opening this PR @kleomedes We just pointed out capitalization of our company's name - I know it's confusing :D

Np! thanks for reviewing this!

File: seda/chain.json [hunk 5]
```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "seda",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.seda.xyz/",
+  "pretty_name": "SEDA",
+  "chain_id": "seda-1",
+  "bech32_prefix": "seda",
```
Comment (JeremyParish69) [inline seda/chain.json:9]: Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: seda/assetlist.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+      "display": "seda",
+      "symbol": "SEDA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/seda/images/seda.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/seda/images/seda.svg"
+      },
+      "coingecko_id": "seda-2",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/seda/images/seda.png",
```
Comment (JeremyParish69) [inline seda/assetlist.json:28]: either the uploaded images have to be visually identical, or these urls need to be in separate objects to show that they're different images.

File: seda/chain.json (hunk 5 above)
Comment (hacheigriega) [reply seda/chain.json:9]: Opened a PR here https://github.com/satoshilabs/slips/pull/1720

Comment (hacheigriega) [reply seda/chain.json:9]: merged.

File: seda/assetlist.json [hunk 7]
```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "seda",
+  "assets": [
+    {
+      "description": "The SEDA Token (SEDA), previously known as Flux Token (FLX), serves three distinct purposes: governance, participation, and network utilization.",
```
Comment (hacheigriega) [inline seda/assetlist.json:6]: Can we add an `extended_description` that reads "SEDA is a standard for modular data transport and querying. Any data type, for all networks."

File: seda/chain.json (hunk 5 above)
Comment (kleomedes) [reply seda/chain.json:9]: thanks @hacheigriega !

**Pull request merged**